js/app.js

```diff
@@ -0,0 +1,114 @@
+;(function() {
+    function sleep(ms) {
+        return new Promise(resolve => setTimeout(resolve, ms));
+    }
+
+    window.addEventListener('DOMContentLoaded', () => {
+
+        const hamburger$ = document
+            .querySelector('#hamburger')
+
+        const hamburgerBackground$ = document
+            .querySelector('.hamburger__overlay-to-close')
+
+        const feedbackOpenButton$ = document
+            .querySelector('.h-nav__feedback')
+
+        const feedbackCloseButton$ = document
+            .querySelector('.feedback-form__close-button')
+
+        const feedbackForm$ = document
+            .querySelector('.feedback-form')
+
+        const feedbackFormInner$ = document
+            .querySelector('.feedback-form__inner')
+
+        const emailOpenButton$ = document
+            .querySelector('.email-screen__input-button')
+
+        const emailForm$ = document
+            .querySelector('.email-form')
+
+        const emailCloseButton$ = document
+            .querySelector('.email-form__close-button')
+
+        const formsBackground$ = document
+            .querySelector('.forms__background')
+
+        const emailFormInner$ = document
+            .querySelector('.email-form__inner')
+
+
+        hamburger$
+            .addEventListener('input', () => {
+                if (hamburger$.checked == true) {
+                    hamburgerBackground$.style.display = "block";
+                    sleep(1).then(() => {
```

Comment:
If you set `0` for `ms` as default value, there is you can use it like `sleep().then(() => {`.